website/static/js/fangorn.js

```diff
@@ -479,6 +481,11 @@ function doItemOp(operation, to, from, rename, conflict) {
         }
         // no need to redraw because fangornOrderFolder does it
         orderFolder.call(tb, from.parent());
+
+        if (operation === OPERATIONS.MOVE) {
```

Comment:
That is the approach that I initially took with osfstorage files, but in the case with external providers a request is getting sent to waterbutler and the file grid is updated with the response. It doesn't seem to go through the osf backend.